Sources/Transaction/Memos/DestinationMemo.swift

```diff
@@ -38,7 +41,12 @@ struct RecoverableDestinationMemo {
             return nil
         }
         let addressHash = DestinationMemoUtils.getAddressHash(memoData: memoData)
-        return DestinationMemo(memoData: memoData, addressHash: addressHash, numberOfRecipients: numberOfRecipients, fee: fee, totalOutlay: totalOutlay)
+        return DestinationMemo(
+            memoData: memoData,
+            addressHash: addressHash,
+            numberOfRecipients: numberOfRecipients,
+            fee: fee,
+            totalOutlay: totalOutlay)
```

Comment:
You typically use double-indentation for parameter lists in actual calls:

```suggestion
                memoData: memoData,
                addressHash: addressHash,
                numberOfRecipients: numberOfRecipients,
                fee: fee,
                totalOutlay: totalOutlay)
```

Reply:
single indentation is the general rule